examples/cors/backend/docker-compose.yaml

```diff
@@ -1,4 +1,4 @@
-version: '2'
+version: '3.7'
```

Comment:
Nit. I didn't mention this on my comment on the prior issue but for consistency with quoting in YAML files can you also change the quoting here from single quote to double quote. ie "3.7". Same for the other occurrences.

Reply:
Gotcha, will update thanks 👍